Library/Homebrew/cmd/home.rb

```diff
@@ -4,11 +4,28 @@
 #:  * `home` <formula>:
 #:    Open <formula>'s homepage in a browser.
 
+require "cli_parser"
+
 module Homebrew
   module_function
 
+  def home_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `home` [<formula>]
+
+        Open <formula>'s homepage in a browser. If no formula is provided,
+        open Homebrew's own homepage in a browser.
+      EOS
+      switch :verbose
```

Comment:
@GauthamGoli What do you think about only adding the `verbose` switch where it results in behaviour or output change?

Reply:
That'd be better and intuitive. Will remove where its redundant.